### DISCUSSION TITLE
一个新的Bot编写者应该注意哪些地方

### DISCUSSION THREAD

**Discussion by tianhan1998** [mamoe/mirai#790] [Q&A] [answered]
你好，我最近才刚开始接触mirai。并申请了一个新的qq作为自己的bot。但是我在编写并调试的过程中发现qq会被冻结。解封需要另一个绑定了银行卡的qq来扫码解封。我想请问一下编写bot需要注意哪些地方。比如不能经常重启？或者q龄高的被冻结几率低？抱歉这个discussion跟mirai没有什么关系。。。。希望能得到一些建议。谢谢

**Comment by Karlatemp** [comment, marked as answer]
注意不要过于频繁重启，过去频繁重启重新登入可能会触发服务器安全规则
每次登入都使用同一设备信息，如果你直接使用core，可以在 BotConfiguration 找到相关api，登入使用不同信息70%会触发安全规则

**Reply by tianhan1998** [reply]
谢谢
